Content.Server/Power/Pow3r/BatteryRampPegSolver.cs

```diff
@@ -165,15 +172,15 @@ private void UpdateNetwork(Network network, PowerState state, float frameTime)
                     if (!battery.Enabled || !battery.CanDischarge || battery.Paused)
                         continue;
 
-                    var scaledSpace = battery.CurrentStorage * frameTime;
+                    var scaledSpace = battery.CurrentStorage / frameTime;
                     var supplyCap = Math.Min(battery.MaxSupply,
                         battery.SupplyRampPosition + battery.SupplyRampTolerance);
                     var supplyAndPassthrough = supplyCap + battery.CurrentReceiving * battery.Efficiency;
 
-                    battery.AvailableSupply = Math.Min(battery.CurrentStorage, supplyAndPassthrough);
+                    battery.AvailableSupply = Math.Min(scaledSpace, supplyAndPassthrough);
                     battery.LoadingNetworkDemand = unmet;
 
-                    battery.MaxEffectiveSupply = Math.Min(battery.CurrentStorage, battery.MaxSupply + battery.CurrentReceiving * battery.Efficiency);
+                    battery.MaxEffectiveSupply = Math.Min(battery.CurrentStorage / frameTime, battery.MaxSupply + battery.CurrentReceiving * battery.Efficiency);
```

Comment:
Thank you for these fixes. Can you verify that these do not affect the light flickering? If not, would you mind submitting these changes in a separate PR so that it's easier to revert in case something goes wrong?

Reply:
Sorry I am still not used to Github so I see these messages randomly. Can you be more specific which changes you are referring to and also how I should go about submitting and updating PRs with changes?

> Thank you for these fixes. Can you verify that these do not affect the light flickering? If not, would you mind submitting these changes in a separate PR so that it's easier to revert in case something goes wrong?